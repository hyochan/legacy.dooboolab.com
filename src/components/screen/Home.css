@import '../../root.css';

/*-------------
  General
-------------*/
.container {
  display: flex;
  flex-direction: column;
  align-items: center;
  max-width: 100vw;
  overflow: hidden;
  white-space: pre-wrap;

  & nav {
    list-style: none;
  }

  & a {
    text-decoration: none;
    cursor: pointer;
    opacity: 0.9;
  }

  & a:hover {
    opacity: 1;
  }

  & hr {
    width: 150px;
    height: 2px;
    background-color: #2196f3;
    border: 0;
    margin-bottom: 80px;
  }

  & section {
    display: flex;
    flex-direction: column;
    align-items: center;
  }
}

.btn {
  color: #fff;
  border-radius: 4px;
  text-transform: uppercase;
  background-color: #2196f3;
  font-weight: 800;
  text-align: center;
}

.btn:disabled {
  color: #fff;
  cursor: not-allowed;
  border-radius: 4px;
  text-transform: uppercase;
  background-color: #ccc;
  font-weight: 800;
  text-align: center;
}

.btn-contact {
  cursor: pointer;
  width: 150px;
  height: 36px;
  margin-top: 30px;
  border-radius: 37.5px;
  background-color: #b13ee5;
  box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.15);

  display: flex;
  justify-content: space-between;
  align-items: center;
  flex-direction: row;
  @media (--desktop) {
    margin-top: 40px;
    width: 200px;
    height: 48px;
    border-radius: 50px;
  }
  & .txt {
    height: 12px;
    margin-left: 42px;
    font-size: 12px;
    line-height: 1;
    color: #ffffff;

    @media (--desktop) {
      height: 18px;
      margin-left: 52px;
      font-size: 18px;
    }
  }
  & .right {
    width: 36px;
    height: 36px;
    margin-top: 30px;
    border-top-right-radius: 37.5px;
    border-bottom-right-radius: 37.5px;
    background-color: #8e2dba;

    display: flex;
    align-self: flex-end;
    justify-content: center;
    align-items: center;

    @media (--desktop) {
      width: 48px;
      height: 48px;
      margin-top: 40px;
      border-top-right-radius: 50px;
      border-bottom-right-radius: 50px;
    }

    & .icn-arrowforward {
      width: 24px;
      height: 24px;
      object-fit: contain;
    }
  }
}

/*----------------
  Hero Section
----------------*/

.hero {
  background-image: linear-gradient(to bottom, #3c96d2, #3ee5ac);
  color: #fff;
  text-align: center;
  min-height: 600px;

  display: flex;
  justify-content: center;
  align-self: stretch;
  & .header {
    margin: 20px;
    padding: 0 30px;
    width: 100%;
    color: #fff;

    display: flex;
    flex: 0 0 auto;
    flex-direction: column;
    justify-content: space-between;
    align-items: center;
    @media (--desktop) {
      flex-direction: row;
    }

    & h2 {
      margin-bottom: 15px;
      @media (--desktop) {
        margin-bottom: 0;
      }
    }

    & nav {
      display: flex;
      & li {
        margin: 0 5px;
        height: 12px;
        font-size: 12px;
        line-height: 1;
        text-align: center;
        @media (--desktop) {
          width: 82px;
          height: 20px;
          font-size: 20px;
        }
        & a {
          color: #ffffff;
        }
      }
      & li:first-child {
        margin-left: 0;
      }
      & li:last-child {
        margin-right: 0;
      }
    }
  }

  & h1 {
    font: bold 24px 'Open Sans', sans-serif;
    margin-top: 48px;
    margin-bottom: 24px;
    width: 790px;
    font-weight: 300;
    line-height: 1.25;
    text-align: center;
    color: #ffffff;
    @media (--desktop) {
      font: 48px 'Open Sans', sans-serif;
      margin-top: 116px;
      font-weight: 300;
    }
  }

  & .body {
    width: 820px;
    background-image: var(--icBgOvals);

    display: flex;
    flex-direction: column;
    align-items: center;

    & h3 {
      font: normal 9px 'Open Sans', sans-serif;
      height: 56px;
      font-family: NotoSansCJKkr-Demi;
      font-weight: 300;
      line-height: 1.56;
      text-align: center;
      color: #ffffff;

      display: flex;
      justify-content: center;
      align-items: center;
      @media (--desktop) {
        font: 18px 'Open Sans', sans-serif;
        font-weight: 300;
      }
    }

    & .img-wrapper {
      width: 60vw;
      margin-top: 44px;
      margin-bottom: 24px;
      @media (--desktop) {
        margin-top: 36px;
        margin-bottom: 60px;
      }
    }

    & a.btn {
      padding: 15px 40px;

      @media (--desktop) {
        padding: 0;
      }
    }
  }
}

/*----------------------
  company Section
----------------------*/

.company {
  margin-top: 120px;
  background-color: #fff;
  align-self: stretch;
  text-align: left;

  display: flex;
  flex-direction: column;
  align-items: flex-start;
  @media (--desktop) {
    align-items: flex-start;
  }

  & .intro {
    align-self: stretch;
    padding: 0 20%;

    display: flex;
    flex-wrap: wrap;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    @media (--desktop) {
      padding: 0 9%;

      flex-direction: row;
      justify-content: space-between;
      align-items: flex-start;
    }

    & .left {
      display: flex;
      flex-direction: column;

      & .title {
        font-size: 10px;
        line-height: 1;
        color: #b03ee5;

        display: flex;
        flex-direction: row;
        align-self: center;

        @media (--desktop) {
          font-size: 18px;
          align-self: flex-start;
        }
      }
      & .header {
        margin-top: 7px;
        height: 48px;
        font-size: 18px;
        line-height: 1.33;
        color: #222222;
        align-self: center;
        text-align: center;

        display: flex;
        flex-direction: row;
        align-items: flex-start;
        justify-content: flex-start;
        align-content: flex-start;

        @media (--desktop) {
          margin-top: 10px;
          width: 100%;
          height: 96px;
          font-size: 36px;
          align-self: flex-start;
          text-align: left;
        }
      }
      & .body {
        margin-top: 12px;
        width: 100%;
        height: 70px;
        font-size: 9px;
        line-height: 1.56;
        color: #a2a2a2;

        display: flex;
        flex-direction: row;
        text-align: center;

        @media (--desktop) {
          margin-top: 24px;
          width: 362px;
          height: 140px;
          font-size: 18px;
          text-align: left;
        }
      }
    }

    & .right {
      display: flex;
      flex-direction: row;
      margin-top: 30px;
      @media (--desktop) {
        margin-top: 60px;
        margin-right: 60px;
      }
      & .img-iphonehand {
        width: 18vw;
        height: 26vw;
      }
    }
  }

  & .philosophy {
    margin-top: 120px;
    padding-left: 0;
    padding-right: 0;
    padding-top: 72px;
    padding-bottom: 72px;
    background-color: #fafafa;
    align-self: stretch;

    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    @media (--desktop) {
      padding-top: 120px;
      padding-bottom: 200px;
      padding-left: 9%;
      padding-right: 9%;
    }

    & .title {
      font-size: 10px;
      line-height: 1;
      color: #b03ee5;

      display: flex;
      flex-direction: row;
      align-self: center;

      @media (--desktop) {
        align-self: flex-start;
        font-size: 18px;
      }
    }

    & .header {
      margin-top: 10px;
      font-size: 18px;
      line-height: 1.33;
      color: #222222;
      align-self: center;

      display: flex;
      flex-direction: row;
      align-items: flex-start;
      justify-content: flex-start;
      align-content: flex-start;

      @media (--desktop) {
        font-size: 36px;
        align-self: flex-start;
      }
    }

    & .body {
      margin-top: 48px;
      width: 100%;
      font-size: 10px;
      line-height: 1.56;
      color: #a2a2a2;

      display: flex;
      flex-direction: column;
      align-items: center;
      font-size: 10px;

      @media (--desktop) {
        font-size: 18px;
        margin-top: 24px;
        flex-direction: row;
        justify-content: center;
      }
      & .card {
        width: 100%;

        display: flex;
        flex-direction: column;
        align-items: center;

        @media (--desktop) {
          align-items: flex-start;
          margin-top: 24px;
        }

        & .title {
          margin-top: 16px;
          height: 14px;
          font-size: 11px;
          line-height: 1.4;
          color: #222222;
          align-self: center;

          @media (--desktop) {
            align-self: flex-start;
            margin-top: 32px;
            height: 28px;
            font-size: 20px;
          }
        }

        & .icn {
          width: 24px;
          height: 24px;

          @media (--desktop) {
            width: 48px;
            height: 48px;
          }
        }

        & .content {
          margin-top: 24px;
          height: 56px;
          font-size: 11px;
          line-height: 1.56;
          color: #a2a2a2;

          @media (--desktop) {
            align-self: flex-start;
            margin-top: 12px;
            height: 28px;
            font-size: 18px;
          }
        }
      }
    }
  }
}

/*--------------------
  Product Section
---------------------*/
.product {
  align-self: stretch;
  margin-top: 30px;
  padding-bottom: 180px;

  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;

  & .content {
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 95vw;
    margin-top: 40px;

    @media (--desktop) {
      margin-top: 100px;
      padding-bottom: 40px;
    }

    & .title {
      height: 18px;
      font-size: 10px;
      line-height: 1;
      color: #b03ee5;

      @media (--desktop) {
        font-size: 18px;
      }
    }
    & .info {
      margin-top: 20px;
      font-size: 16px;
      line-height: 1;
      text-align: center;

      @media (--desktop) {
        margin-top: 22px;
        font-size: 32px;
      }
    }
    & .description {
      font-size: 10px;
      line-height: 1.56;
      text-align: center;
      color: #a2a2a2;

      @media (--desktop) {
        font-szie: 18px;
      }

      & .img-iphone {
        padding-top: 25px;
        width: 80%;
        object-fit: contain;

        @media (--desktop) {
          padding-top: 56px;
          font-szie: 32px;
        }
      }
    }
    & .btns {
      margin-top: 20px;
      height: 24px;
      margin-bottom: 40px;

      display: flex;
      flex-wrap: wrap;
      justify-content: center;

      @media (--desktop) {
        height: 48px;
        margin-top: 36px;
      }
      & a {
        margin: 8px;
        width: 160px;
        height: 32px;
        opacity: 0.54;
        border-radius: 100px;
        background-color: #b13ee5;
        box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.15);
        font-size: 10px;
        line-height: 1;
        text-align: center;
        color: #ffffff;

        display: flex;
        align-items: center;
        justify-content: center;

        @media (--desktop) {
          font-size: 18px;
          height: 48px;
          width: 200px;
        }
      }
    }
  }
  & .right {
    z-index: 100;
    width: 48px;
    height: 48px;
    display: flex;
    justify-content: center;
    align-items: center;
    cursor: pointer;
    border-radius: 50px;
    background-color: #8e2dba;
    box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.15);
    & .icn-arrowforward {
      width: 24px;
      height: 24px;
      object-fit: contain;
    }
  }
}

/*---------------------
  Contact Section
---------------------*/
.contact {
  margin-top: 80px;
  margin-bottom: 100px;

  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  align-self: stretch;

  @media (--desktop) {
    margin-top: 120px;
    margin-bottom: 160px;
  }

  & .title {
    font-size: 11px;
    line-height: 1;
    color: #b03ee5;

    display: flex;
    justify-content: center;
    align-items: center;

    @media (--desktop) {
      font-size: 18px;
    }
  }

  & .header {
    margin-top: 10px;
    font-size: 18px;
    line-height: 1.33;
    text-align: center;
    color: #222222;

    display: flex;
    justify-content: center;
    align-items: center;

    @media (--desktop) {
      font-size: 36px;
    }
  }

  & .description {
    margin-top: 24px;
    font-size: 10px;
    line-height: 1.56;
    text-align: center;
    color: #a2a2a2;

    display: flex;
    justify-content: center;
    align-items: center;

    @media (--desktop) {
      font-size: 18px;
    }
  }
}

/*-------------
  Footer
-------------*/

.footer {
  align-self: stretch;
  color: #fff;
  background-color: rgb(57, 61, 122);
  text-align: center;
  height: 48px;
  padding: 0;

  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;

  @media (--desktop) {
    height: 96px;
  }

  & p {
    font-size: 11px;
    line-height: 1;
    text-align: center;
    color: #ffffff;

    @media (--desktop) {
      font-size: 20px;
    }
  }
}
