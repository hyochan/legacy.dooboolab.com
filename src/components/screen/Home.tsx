import './Home.css';

import { Link, Redirect, Route } from 'react-router-dom';
import React, { Component } from 'react';
import { inject, observer } from 'mobx-react';

import Carousel from 'nuka-carousel';
import CooniBtn from '@shared/CooniBtn';
import FontAwesome from 'react-fontawesome';

export class Home extends Component<any, any> {
  constructor(props) {
    super(props);
    this.state = {};
  }

  public render() {
    const { getString } = this.props.store.locale;
    return (
      <div className='container'>
        <section id='hero' className='hero'>
          <header className='header'>
            <h2
              style={
                this.props.store.isDesktop
                  ? { marginLeft: '35px' }
                  : { marginLeft: 0 }
              }
            >
              <img
                className='logo'
                src={require('@assets/icons/logo.png')}
                srcSet={`
                  ${require('@assets/icons/logo@2x.png')} 2x,
                  ${require('@assets/icons/logo@3x.png')} 3x
                `}
              />
            </h2>
            <nav
              style={
                this.props.store.isDesktop
                  ? { marginRight: '35px' }
                  : { marginRight: 0 }
              }
            >
              <li>
                <a href='#company'>{getString('INTRO')}</a>
              </li>
              <li>
                <a href='#product'>{getString('PRODUCT')}</a>
              </li>
              <li>
                <a href='#member'>{getString('MEMBER')}</a>
              </li>
              <li>
                <a href='#contact'>{getString('CONTACT')}</a>
              </li>
            </nav>
          </header>
          <h1>
            {getString('FLEXIBLE_INNOVATION')}
            <br />
            {getString('START_WITH_DOOBOOLAB')}
          </h1>
          <div className='body'>
            <h3 style={{ lineHeight: 0.8 }}>{getString('HEADER_TXT')}</h3>
            <a className='btn-contact' href='mailto:dooboolab@gmail.com'>
              <div className='txt'>{getString('CONTACT')}</div>
              <div className='right'>
                <img
                  className='icn-arrowforward'
                  src={require('@assets/icons/icn-arrowforward.png')}
                  srcSet={`
                    ${require('@assets/icons/icn-arrowforward@2x.png')} 2x,
                    ${require('@assets/icons/icn-arrowforward@3x.png')} 3x
                  `}
                />
              </div>
            </a>
            <img
              className='img-wrapper'
              alt='Null'
              src={require('@assets/icons/img-iphone.png')}
              srcSet={`
                ${require('@assets/icons/img-iphone.png')} 2x,
                ${require('@assets/icons/img-iphone.png')} 3x
              `}
            />
          </div>
        </section>

        <section id='company' className='company'>
          <div className='intro'>
            <div className='left'>
              <p className='title'>{getString('ABOUT_DOOBOOLAB')}</p>
              <p className='header'>{getString('ABOUT_DOOBOOLAB_HEADER')}</p>
              <p className='body'>{getString('ABOUT_DOOBOOLAB_BODY')}</p>
            </div>
            <div className='right'>
              <img
                className='img-iphonehand'
                alt='Null'
                src={require('@assets/icons/img-iphonehand.png')}
                srcSet={`
                  ${require('@assets/icons/img-iphonehand.png')} 2x,
                  ${require('@assets/icons/img-iphonehand.png')} 3x,
                `}
              />
            </div>
          </div>
          <div className='philosophy'>
            <p className='title'>{getString('PHILOSOPHY_DOOBOOLAB')}</p>
            <p className='header'>{getString('PHILOSOPHY_DOOBOOLAB_HEADER')}</p>
            <div className='body'>
              <div className='card'>
                <img
                  className='icn'
                  src={require('@assets/icons/icn-ux.png')}
                  srcSet={`
                    ${require('@assets/icons/icn-ux.png')} 2x,
                    ${require('@assets/icons/icn-ux.png')} 3x,
                  `}
                />
                <p className='title'>{getString('PLANNING_AND_DESIGN')}</p>
                <p className='content'>
                  {getString('PLANNING_AND_DESIGN_BODY')}
                </p>
              </div>
              <div
                className='card'
                style={
                  this.props.store.isDesktop ? { marginLeft: '40px' } : null
                }
              >
                <img
                  className='icn'
                  src={require('@assets/icons/icn-dev.png')}
                  srcSet={`
                    ${require('@assets/icons/icn-dev.png')} 2x,
                    ${require('@assets/icons/icn-dev.png')} 3x,
                  `}
                />
                <p className='title'>{getString('DEVELOPMENT')}</p>
                <p className='content'>{getString('DEVELOPMENT_BODY')}</p>
              </div>
              <div
                className='card'
                style={
                  this.props.store.isDesktop ? { marginLeft: '40px' } : null
                }
              >
                <img
                  className='icn'
                  src={require('@assets/icons/icn-frontend.png')}
                  srcSet={`
                    ${require('@assets/icons/icn-frontend.png')} 2x,
                    ${require('@assets/icons/icn-frontend.png')} 3x,
                  `}
                />
                <p className='title'>{getString('DESIGN')}</p>
                <p className='content'>{getString('DESIGN_BODY')}</p>
              </div>
            </div>
          </div>
        </section>

        <section id='product' className='product'>
          <Carousel wrapAround={true} width='97vw'>
            <div className='content'>
              <p className='title'>{getString('DOOBOOLAB_WORK')} 01</p>
              <p className='info'>BooKoo</p>
              <div className='btns'>
                <a href='https://play.google.com/store/apps/details?id=com.dooboolab.bookoo'>
                  Android
                </a>
                <a href='https://itunes.apple.com/us/app/bookoo/id1276062364?l=ko&ls=1&mt=8'>
                  IOS
                </a>
              </div>
              <p className='description'>
                <img
                  className='img-iphone'
                  src={require('@assets/images/BooKooRelease.png')}
                />
              </p>
            </div>
            <div className='content'>
              <p className='title'>{getString('DOOBOOLAB_WORK')} 02</p>
              <p className='info'>COONI</p>
              <div className='btns'>
                <a href='https://play.google.com/store/apps/details?id=com.cooni.app'>
                  Android
                </a>
                <a href='https://itunes.apple.com/us/app/cooni/id1315512929?ls=1&mt=8'>
                  IOS
                </a>
              </div>
              <p className='description'>
                <img
                  className='img-iphone'
                  src={require('@assets/images/CooniRelease.png')}
                />
              </p>
            </div>
            <div className='content'>
              <p className='title'>{getString('DOOBOOLAB_WORK')} 03</p>
              <p className='info'>foodieon</p>
              <div className='btns'>
                <a href='https://play.google.com/store/apps/details?id=com.foodieon.app'>
                  Android
                </a>
                <a href='https://itunes.apple.com/kr/app/foodieon-%ED%91%B8%EB%94%94%EC%98%A8/id1424869463?mt=8'>
                  IOS
                </a>
              </div>
              <p className='description'>
                <img
                  className='img-iphone'
                  src={require('@assets/images/FoodieonRelease.png')}
                />
              </p>
            </div>
          </Carousel>
        </section>

        <section id='member' className='member'>
          <p className='title'>{getString('DOOBOOLAB_MEMBER')}</p>
          {this.renderMembers()}
          <button className='btn'>WHO's NEXT?</button>
        </section>

        <section id='contact' className='contact'>
          <p className='title'>{getString('CONTACT')}</p>
          <p className='header'>
            Who’s next?
            <br />
            {getString('FREE_TALK')}
          </p>
          <p className='description'>{getString('FREE_TALK_DESCRIPTION')}</p>
          <a className='btn-contact' href='mailto:dooboolab@gmail.com'>
            <div className='txt'>{getString('CONTACT')}</div>
            <div className='right'>
              <img
                className='icn'
                src={require('@assets/icons/icn-arrowforward.png')}
                srcSet={`
                  ${require('@assets/icons/icn-arrowforward.png')} 2x,
                  ${require('@assets/icons/icn-arrowforward.png')} 3x,
                `}
              />
            </div>
          </a>
        </section>

        <footer className='footer'>
          <p>©2017 Dooboolab, All Rights Reserved</p>
        </footer>
      </div>
    );
  }

  private renderMembers() {
    return (
      <div className='content'>
        {/* Hyo Chan Jang */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/hyochan.jpg')} />
          <p className='name'>hyochan</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/hyochan-jang-8723a5108/'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://twitter.com/dooboolab'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-twitter' name='twitter' />
              </i>
            </a>
            <a href='https://github.com/dooboolab'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
        {/* Mars */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/mars.jpg')} />
          <p className='name'>mars</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/mars-kim-6633b4187/'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://github.com/marsinearth'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
        {/* Geoseong */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/geoseong.jpg')} />
          <p className='name'>geoseong</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/tae-seong-park-0699b348'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://github.com/geoseong'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
        {/* Yong Pil */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/yongpil.jpg')} />
          <p className='name'>yongpil</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/yongpil-moon-57877b147/'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://github.com/YongPilMoon'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
        {/* Dong Kyun */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/dongkyun.jpg')} />
          <p className='name'>gordon</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/dong-kyun-ko-5a2819171/'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://github.com/godon019'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
        {/* Jong Taek */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/jongtaek.png')} />
          <p className='name'>jongtaek</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/jongtaek-choi-259796117/'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://github.com/jongtaekChoi'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
        {/* youngsoo */}
        <div className='box'>
          <img className='pic' src={require('@assets/images/youngsoo.png')} />
          <p className='name'>youngsoo</p>
          <p className='position'>developer</p>
          {/* <p className='introduction'></p> */}
          <div className='underline' />
          <div className='sns'>
            <a href='https://www.linkedin.com/in/youngsu-han/'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-linkedin' name='linkedin' />
              </i>
            </a>
            <a href='https://twitter.com/pEy69B9Aest8VRT'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-twitter' name='twitter' />
              </i>
            </a>
            <a href='https://github.com/heyman333'>
              <i aria-hidden='true'>
                <FontAwesome className='fa-github' name='github' />
              </i>
            </a>
          </div>
        </div>
      </div>
    );
  }
}

const defaultHome = inject('store')(observer(Home));
export default defaultHome;
