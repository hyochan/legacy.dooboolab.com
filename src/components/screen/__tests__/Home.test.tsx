import React from 'react';
import { shallow, render } from 'enzyme';

import Store from '../../../stores/appStore';
import { Home } from '../Home';

const props = {
  store: new Store(),
  navigation: {
    navigate: jest.fn(),
  },
  history: {
    push: jest.fn(),
  },
};

// test for the container page in dom
describe('Tab1 page DOM rendering test', () => {
  const page = shallow(
    <Home { ...props }/>,
  );

  it('Home page has to be matched to snapshot', () => {
    expect(page).toMatchSnapshot();
  });
});
