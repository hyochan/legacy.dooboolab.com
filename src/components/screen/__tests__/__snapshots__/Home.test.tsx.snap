// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab1 page DOM rendering test Home page has to be matched to snapshot 1`] = `ShallowWrapper {}`;
